### PR TITLE
added deletion prompt for installed runners

### PR DESCRIPTION
**Pull request by Termuellinator**

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -35,7 +35,61 @@ InstallWineRunner() {
   curl -sL "$url" | tar xfzv - -C "$dest_path"
   install_complete=true
   echo "Installation completed"
-  RestartLutrisCheck
+  Wannadelete
+}
+
+Wannadelete() {
```
Comment (richardtatum) [inline update.sh]: Lets rename this to `DeleteRestartPrompt()`, to keep it descriptive.

File: update.sh [hunk 2]
```diff
@@ -35,7 +35,61 @@ InstallWineRunner() {
...
+
+DeleteRunnersCheck() {
+    echo "Installed runners:"
+    installed_versions=($(ls -d "$base_path"/*/))
+    for((i=0;i<${#installed_versions[@]};i++)); do
+        inumber=$(("$i" + 1))
+        folder=$(echo "${installed_versions[i]}" | rev | cut -d/ -f2 | rev)
+        echo "$inumber. $folder"
+    done
+    echo -n "Please choose an option to remove [1-${#installed_versions[@]}]:"
```
Comment (richardtatum) [inline update.sh:61]: Can we add an empty line here to keep styling similar to the other `choose option` function?

```
echo ""
echo -n "Please choose an option to remove [1-${#installed_versions[@]}]:"
```

File: update.sh (hunk 1 above)
Comment (Termuellinator) [reply update.sh]: you're right, that sounds better. It can be hard to come up with good function names ;)

File: update.sh (hunk 2 above)
Comment (Termuellinator) [reply update.sh:61]: Didn't catch that, thanks! :)

**Pull request merged**